tests/data/test_tinystories.py

```diff
@@ -80,17 +80,12 @@ def test_tinystories_datamodule(tmp_path):
     actual = tree_map(torch.Tensor.tolist, list(tr_dataloader))
     # there is 1 sample per index in the data (13)
```

Comment:
This comment is incorrect now, what motivates the 8 samples now?

The previous assertion also was close to what karpathy implemented in his repo. Whatever this implements is no longer similar

Reply:
It's due to the shuffling logic in combination with `drop_last=True`. We could set `drop_last=False` in the datamodule. Or we could try to restore the previous behavior in LitData.

Reply:
Addressed in LitData and #1609